crates/protocol/protocol/src/predeploys.rs

```diff
@@ -12,9 +12,82 @@ pub struct Predeploys;
 
 impl Predeploys {
     /// List of all predeploys.
-    pub const ALL: [Address; 1] = [Self::L2_TO_L1_MESSAGE_PASSER];
+    pub const ALL: [Address; 18] = [
+        Self::WETH9,
+        Self::L2_CROSS_DOMAIN_MESSENGER,
+        Self::L2_STANDARD_BRIDGE,
+        Self::SEQUENCER_FEE_VAULT,
+        Self::OP_MINTABLE_ERC20_FACTORY,
+        Self::GAS_PRICE_ORACLE,
+        Self::GOVERNANCE_TOKEN,
+        Self::L2_ERC721_BRIDGE,
+        Self::L1_BLOCK_INFO,
+        Self::L2_TO_L1_MESSAGE_PASSER,
+        Self::OP_MINTABLE_ERC721_FACTORY,
+        Self::PROXY_ADMIN,
+        Self::BASE_FEE_VAULT,
+        Self::L1_FEE_VAULT,
+        Self::SCHEMA_REGISTRY,
+        Self::EAS,
+        Self::OPERATOR_FEE_VAULT,
+        Self::CROSS_L2_INBOX,
+    ];
+
+    /// The WETH9 predeploy address.
```

Comment:
would be really nice if we could link to the solidity code for each one, like this
```suggestion
    /// The WETH9 predeploy address.
    /// `WETH9.sol`: <https://optimistic.etherscan.io/address/0x4200000000000000000000000000000000000006#code#L1>
```

Reply:
Good thinking. Will do when I'm at my desk